kubernetes/helm_charts/core/content/values.j2

```diff
@@ -5,31 +5,31 @@ imagepullsecrets: {{ imagepullsecrets }}
 dockerhub: {{ dockerhub }}
 
 env:
-  javaoptions: {{content_java_mem_limit|default('-Xmx600m')}}
+  javaoptions: {{uci_outbound_java_mem_limit|default('-Xmx600m')}}
```

Comment:
Why are we editing `kubernetes/helm_charts/core/content/values.j2`?

Reply:
Changes were made to wrong file, reverted the changes.